### PR TITLE
WEBEU2 - Preprocessing II - James Eneh 🇳🇬

### PR DESCRIPTION
**Pull request by erozonachi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,29 +4,29 @@ Fun Bus is a travel agency looking for some help on their website.  They want a
 
 ## Task 1: Set Up The Project With Git
 
-- [ ] Create a forked copy of this project.
-- [ ] Add your project manager as collaborator on Github.
-- [ ] Clone your OWN version of the repository (Not Lambda's by mistake!).
-- [ ] Create a new branch: git checkout -b `<firstName-lastName>`.
-- [ ] Implement the project on your newly created `<firstName-lastName>` branch, committing changes regularly.
-- [ ] Push commits: git push origin `<firstName-lastName>`.
+- [x] Create a forked copy of this project.
```
Comment (maximesalomon) [inline README.md:7]: Good job using the README!

File: index.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0;">
```
Comment (maximesalomon) [inline index.html:6]: 👍

File: index.html [hunk 3]
```diff
@@ -15,55 +16,79 @@
...
-      <div class="destination">
-        <h4>Island Getaway</h4>
-        <p>Expedition excursion design excursion fun, clean simple organized WordPress Travel colorful webdesign. Traveler blogger website design expedition clean excursion traveling.</p>
-        <div class="btn">Sign Me Up!</div>
-      </div>
-    </section>
+    </nav>
+  </header>
+  <div class="container home">
+    <main>
```
Comment (maximesalomon) [inline index.html:33]: 👍

File: less/index.less [hunk 4]
```diff
@@ -1 +1,8 @@
 // Follow the order in the readme
+@import "variables";
```
Comment (maximesalomon) [inline less/index.less:2]: import import import import :)

File: less/mixins.less [hunk 5]
```diff
@@ -1 +1,18 @@
...
+  text-align: center;
+  border: none;
+  outline: none;
+  border-radius: .5em;
+  background-color: @bg-color;
+  color: @text-color;
+  cursor: pointer;
+}
+
+.animate-text(@name, @duration) {
```
Comment (maximesalomon) [inline less/mixins.less:15]: Noiice!

File: less/variables.less [hunk 6]
```diff
@@ -11,4 +11,7 @@
 @silver: #C0C0C0;
 @shark: #212529;
 @eastern-blue: #17A2B8;
-@sandy-beach: #FFEBCD;
\ No newline at end of file
+@sandy-beach: #FFEBCD;
+
+// Devices
+@mobile: ~"(max-width: 500px)";
```
Comment (maximesalomon) [inline less/variables.less:17]: 👍

**Review by maximesalomon — COMMENTED** [review]
Great job with this project James 🔥

Your code looks clean and it shows that you took your time

MPV = ✅
* Imported LESS Files ✅
* Implemented the navigation, header, and button ✅
* Desktop ✅
* Used a parametric mixin for your buttons ✅
* Used escaping to create a @mobile variable ✅
* Mobile ✅

Stretch Goals =
* Created an animation mixin using parametric mixins ✅

There are a few missing pieces to make it perfect but I think you needed just a bit more time. Great job overall!

Try to follow the git commit guidelines - https://chris.beams.io/posts/git-commit/

You got this 💪

**Comment by erozonachi** [comment]
Feedback well received. Thanks, @maximesalomon!😎

**Pull request merged**